Header header
sensor_msgs/LaserScan echo_1
sensor_msgs/LaserScan echo_2
int32[] object_mask
